Project.toml

```diff
@@ -8,6 +8,7 @@ AbstractAlgebra = "c3fe647b-3220-5bb0-a1ea-a7954cac585d"
 AlgebraicSolving = "66b61cbe-0446-4d5d-9090-1ff510639f9d"
 DocStringExtensions = "ffbed154-4ef7-542d-bbb7-c09d3a79fcae"
 GAP = "c863536a-3901-11e9-33e7-d5cd0df7b904"
+Glob = "c27321d9-0574-5035-807b-f59d2c89b15c"
```

Comment:
Is glob stable? Supported? Core Julia?

Reply:
What is "Core Julia"? It is not a stdlib if that's what you mean, rather https://github.com/vtjnash/Glob.jl . It is stable in that its interface has been basically unchanged for the past couple years. Its author is one of the primary Julia developer. I just asked very recently if he could make a new release and he did so immediately.

Anyway, we could also rewrite the code to just use built-in Julia stuff, it'd not be hard. But maybe before we think about that, we should first see if the rest of this PR is something we want to go forward with